opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/authorization/oauth2/xs2a-oauth2-step.bpmn20.xml

```diff
@@ -23,6 +23,15 @@
       <conditionExpression xsi:type="tFormalExpression">#{xs2aConsentInfo.isOauth2TokenAvailableAndReadyToUse(CONTEXT)}</conditionExpression>
     </sequenceFlow>
     <sequenceFlow id="sid-cf198380-6fed-408b-8641-0a918a74a2db" sourceRef="oauth2ExchangeCodeToToken" targetRef="sid-e0dbf285-c6c2-4571-ac5e-9038fcbf8448"/>
+    <sequenceFlow id="sid-39bca444-9c5a-4432-9662-e41c0b2e1231" name="Embedded pre Auth required" sourceRef="checkThatOauth2TokenIsPresentAndNotExpired" targetRef="xs2aAskForPassword">
+      <conditionExpression xsi:type="tFormalExpression">#{xs2aConsentInfo.isEmbeddedPreAuthNeeded(CONTEXT)}</conditionExpression>
+    </sequenceFlow>
+    <serviceTask id="embeddedOauth2ExchangeCodeToToken" name="Get Embedded Access Token" flowable:async="true" flowable:exclusive="true" flowable:delegateExpression="${xs2aEmbeddedPreAuthorization}" flowable:triggerable="false" isForCompensation="false"/>
```

Comment:
This element name obviously is not `Get Embedded Access Token`, but rather `Exchange password (embedded pre-step) to token`